pkg/storage/client_raft_test.go

```diff
@@ -1622,18 +1622,8 @@ func TestReportUnreachableHeartbeats(t *testing.T) {
 	mtc := startMultiTestContext(t, 3)
 	defer mtc.Stop()
 
-	// Replicate the range onto all three stores
 	mtc.replicateRange(1, 1, 2)
 
-	// Send a single increment to ensure the range is up.
```

Comment:
I'm not sure the way `replicateRange` waiting for the replica to be present on all of the target stores is equivalent. The increment-and-wait-for-values approach guarantees the replicas are caught up to a specific spot which might have been necessary for some code path this test was exercising. Or it could have been copy-pasta.